.github/workflows/test-pr.yml

```diff
@@ -99,7 +99,7 @@ jobs:
     name: 'Build and Test KEVM concrete execution'
     needs: kevm-pyk-code-quality-checks
     runs-on: [self-hosted, linux, normal]
-    timeout-minutes: 45
+    timeout-minutes: 60
     steps:
       - name: 'Check out code'
         uses: actions/checkout@v3
```

Comment:
Are these changes in timeout still needed? Seems like they're leftover from before?

Reply:
Removed these changes.